front-end/src/app/reports/transactions/transaction-list/select-report-dialog/select-report-dialog.component.ts

```diff
@@ -56,4 +56,8 @@ export class SelectReportDialogComponent implements OnInit {
     this.transaction = undefined;
     this.selectReportDialog?.nativeElement.close();
   }
+
+  get electContrib(): string {
```

Comment:
It's a little tricky following the use and what's defined with these getters.  What do you think of these naming changes:
```
  // Label for action the user is trying to perform
  get actionLabel(): string {
    return ReattRedesUtils.isReattribute(this.type) ? 'reattribute' : 'redesignate';
  }

  // URL parameter to set the action type
  get urlParameter(): string {
    return ReattRedesUtils.isReattribute(this.type) ? 'reattribution' : 'redesignation';
  }

  // Label for the target of the action
  get actionTargetLabel(): string {
    return ReattRedesUtils.isReattribute(this.type) ? 'contributor' : 'election';
  }
```

Reply:
Good idea. Updated them with that. Also updated the spec to cover these a bit more fully.